### DISCUSSION TITLE
Iran map with number of killed people on it

### DISCUSSION THREAD

**Discussion by mshariat121** [WeAreMahsaAmini/Core#72] [Ideas]
We can create a map of Iran that show number of killed people on it like bubble chart. Also, we can add list of important places to protest in each city when we hover on each o them, it comes up and the places are linked to google map

**Comment by mrnobud** [comment]
Like this idea, 
We should include the date they got killed as well, 
We can ask people to contribute and add to it for all previous protests or executions
